src/logic/live.py

```diff
@@ -37,6 +37,13 @@ def classify_note(row):
         return NoteType.SLIDE
 
 
+def classify_note_vectorized(row):
+    return np.choose(row.type - 3, [
```

Comment:
Can run a test on all charts except WITCH to cross check with `classify_note`?

Reply:
I've added tests for this in [d55101e](https://github.com/deresute-tools/deresute-tools/pull/17/commits/d55101e4bcf453ed416cc9500262b3f77d93af15). The tests passed.